requirements.txt

```diff
@@ -1,7 +1,7 @@
 git+https://github.com/rwbarton/django.git@59f1fc4c7b65d23fceffd5c87e2b6ea350030c56
 GitPython==0.3.2.1
 Markdown==2.1.1
-Pillow==2.8.2
+Pillow>=2.8.1
```

Comment:
@timabbott Actually, I just remembered why I added this constraint -- it seems that when I install Pillow form pip it gets installed without freetype support and I get the following error when running `./tools/emoji_dump/build_emoji`: `The _imagingft C module is not installed`. If I installed Pillow from OpenBSD's packages, then it runs fine, but as you can guess the current version is 2.8.1.